cmd/eno-controller/main.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/go-logr/zapr"
```

Comment:
Shouldn't the changes in this file be in the reconciler instead?

Reply:
lol 😭